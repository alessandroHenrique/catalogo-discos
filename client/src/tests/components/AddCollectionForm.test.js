import ColletionForm from '../../components/CollectionForm/AddCollectionForm';
import React from 'react';
import renderer from 'react-test-renderer';


describe('AddCollectionForm', () => {
  it('should render add collection form', () => {

    const component = renderer.create(
      <ColletionForm
        handleAddCollection={() => {}}
      />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})
