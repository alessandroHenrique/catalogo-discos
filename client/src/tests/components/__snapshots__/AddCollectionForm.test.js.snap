// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddCollectionForm should render add collection form 1`] = `
<form
  className="add-option"
  onSubmit={[Function]}
>
  <input
    className="add-option__input"
    onChange={[Function]}
    type="text"
    value=""
  />
  <button
    className="button btn-secondary no-hover"
    disabled={true}
    type="submit"
  >
    Add Collection
  </button>
</form>
`;
